hybridse/src/node/expr_node.cc

```diff
@@ -732,6 +733,16 @@ ExprListNode* ExprListNode::ShadowCopy(NodeManager* nm) const {
     return list;
 }
 
+Status ExprListNode::InferAttr(ExprAnalysisContext* ctx) {
```

Comment:
not inferred because `resolve_fn_and_attr` skip if `expr_type = kExprList`
will this a problem @jingchen2222